Gordon360/Controllers/NewsController.cs

```diff
@@ -64,7 +71,9 @@ public ActionResult<string> GetImage(int newsID)
                 return NotFound();
             }
 
-            return Ok(ImageUtils.RetrieveImageFromPath(result.Image));
+            var imagePath = Path.Combine(_webHostEnvironment.ContentRootPath, "browseable", "uploads", Path.GetFileName(result.Image));
+
+            return Ok(ImageUtils.RetrieveImageFromPath(imagePath));
```

Comment:
Since this is only used on GO, we can update this to return the URL same as above.